srcpkgs/goodvibes/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'goodvibes'
+pkgname=goodvibes
+version=0.3.7
+revision=1
+wrksrc="goodvibes-v${version}"
+build_style=gnu-configure
+hostmakedepends="automake gettext-devel pkg-config glib-devel"
+makedepends="dconf-devel glib-devel gst-plugins-base1-devel gtk+3-devel
+ libkeybinder3-devel libsoup-devel gettext-devel"
```

Comment:
Any reason why you put `gettext-devel` and `glib-devel` in both `hostmakedepends` / `makedepends` ?

Reply:
They contain some binaries that are executed while compiling. But they are probably not needed for makedepends, at least gettext not. Will recheck it.

Reply:
I removed them form makedepends since they are not needed there.